task-sdk/src/airflow/sdk/log.py

```diff
@@ -90,9 +91,10 @@ def logger_name(logger: Any, method_name: Any, event_dict: EventDict) -> EventDi
 
 
 def redact_jwt(logger: Any, method_name: str, event_dict: EventDict) -> EventDict:
+    jwt_pattern = r"eyJ[\.A-Za-z0-9-_]*"
     for k, v in event_dict.items():
-        if isinstance(v, str) and v.startswith("eyJ"):
-            event_dict[k] = "eyJ***"
+        if isinstance(v, str):
+            event_dict[k] = re.sub(jwt_pattern, "eyJ***", v)
```

Comment:
This is called often enough that we should pre-compile the regex globally/at module scope

Reply:
Yeah thats a good call

Reply:
Handled in 32b774def4492e07b156807c56bc13aae3836324